packages/lesswrong/components/questions/Answer.tsx

```diff
@@ -122,9 +124,12 @@ const styles = (theme: ThemeType): JssStyles => ({
   },
 })
 
-const Answer = ({ comment, post, classes }: {
+export const ABRIDGE_COMMENT_COUNT = 500;
```

Comment:
Does this end up getting used anywhere? I don't see it referenced in the file where it came from (`AnswerCommentsList`)

Reply:
Not used; got moved by refactor and them made-unused by next refactor.